README.md

```diff
@@ -337,11 +334,6 @@ the process's current working directory if no `package.json` is found.
 
 If you are interesting in writing a resolver, see the [spec](./resolvers/README.md) for more details.
```

Comment:
@SukkaW 

I just copied this file from https://github.com/import-js/eslint-plugin-import/blob/main/resolvers/README.md, we'll need to write our own for `v3` resolver.